src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -763,7 +779,8 @@ def update_connectedk8s(cmd, instance, tags=None):
 
 
 def update_agents(cmd, client, resource_group_name, cluster_name, https_proxy="", http_proxy="", no_proxy="", proxy_cert="",
-                  disable_proxy=False, kube_config=None, kube_context=None, auto_upgrade=None):
+                  disable_proxy=False, kube_config=None, kube_context=None, auto_upgrade=None, container_registry_repository=None,
+                  container_registry_username=None, container_registry_password=None, default_registry=None, anonymous_pull=None):
```

Comment:
The parameter 'anonymous_pull' is irrelevant for non pcr scenarios. So would be better to prefix it with 'container_registry'

Reply:
ack